src/review/index.ts

```diff
@@ -14,7 +13,7 @@ export const review = async (
   yargs: ReviewArgs,
   files: ReviewFile[],
   openAIApiKey: string
-): Promise<void> => {
+): Promise<string | undefined> => {
```

Comment:
So CR_GPT suggested we return strings only so in lines 30-33 when we don't return undefined. But I think we might want to return undefined, since if there are no files to review, we don't want to comment anything. 
Open for a challenge tho!

Reply:
Yeah that makes sense !